isites_migration/templates/isites_migration/index.html

```diff
@@ -20,33 +23,66 @@
 
 <main>
     <div class="content">
-        {% if keywords %}
-        <form class="form-inline" method="POST" action="{% url 'isites_migration:index' %}">
-            {% csrf_token %}
-            <div class="form-group">
-                <select id="keyword" name="keyword" class="form-control">
-                    <option value="">Select a Course iSite</option>
-                    {% for keyword in keywords %}
-                    <option value="{{ keyword }}">{{ keyword }}</option>
+
+        <div class="alert alert-warning" role="alert" hidden="">
+          This Canvas course is not associated with any Course iSites.
+        </div>
+
+        <div class="alert alert-info alert-dismissible" role="alert">
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">×</span></button>
+          This migration will include files (e.g. pdfs, Word documents, etc.) and text box content (e.g. iSites text areas). It will not include web links, single images, students' submitted dropbox content, videos from the Video Publishing Tool, lecture videos, discussions, student-generated content (dropbox, video submissions, quizzes) and web links (including linked YouTube videos in Video Publishing Tool.)
+          <br />&nbsp;<br />
+          iSites listed here are those that you’ve been associated with in a teaching staff role. All the imported material can be found in the <a href="#" class="alert-link">Files</a> area of this Canvas course site. The folder named with the imported iSite’s keyword is set as unpublished, you will have to publish it manually by selecting the cloud icon. You can also add these files directly to your Canvas <a href="#" class="alert-link">Pages</a>, <a href="#" class="alert-link">Announcements</a>, etc… using the File Browser, shown in the right hand panel when you enter into edit mode in these tools.
```

Comment:
I think the comma in "as unpublished, you will" should probably be a semicolon, we can check with Jill.